packages/react-components/react-search/docs/Spec.md

```diff
@@ -2,23 +2,79 @@
 
 ## Background
 
-_Description and use cases of this component_
+`SearchBox` is a text input field abstracting `<input type="search" />`.
 
 ## Prior Art
```

Comment:
Are there any key takeaways from the prior art research? Interesting things that most other searchboxes do but v8/v0 don't, any interesting cases we would want to consider for ours?